src/libraries/tests.proj

```diff
@@ -19,7 +19,7 @@
 
   <ItemGroup>
     <!-- We currently only test with C# projects. -->
-    <ProjectReference Include="$(MSBuildThisFileDirectory)*\tests\**\*.Tests.csproj"
```

Comment:
@safern won't this influence other platforms as well? Maybe we want to add a special clause for `TargetsMobile`?